src/background.js

```diff
@@ -397,6 +397,8 @@ async function areAllStringsTranslated () {
   const resp = await fetch(enMessagesPath);
   const enMessages = await resp.json();
 
+  // TODO: Check Pontoon for available translations instead of checking
```

Comment:
Can you comment out the entire `areAllStringsTranslated()` function? It's breaking Travis CI checks.